source/module_base/parallel_global.cpp

```diff
@@ -176,39 +176,44 @@ void Parallel_Global::read_mpi_parameters(int argc,char **argv)
 	// get the rank --> GlobalV::MY_RANK
 	MPI_Comm_size(MPI_COMM_WORLD,&GlobalV::NPROC);
     MPI_Comm_rank(MPI_COMM_WORLD, &GlobalV::MY_RANK);
+    MPI_Comm shmcomm;
+    MPI_Comm_split_type(MPI_COMM_WORLD, MPI_COMM_TYPE_SHARED, 0, MPI_INFO_NULL, &shmcomm);
```

Comment:
add some explanations?

Reply:
@mohanchen Sure, would you see the latest commit.